tests/pagai/integration/conftest.py

```diff
@@ -0,0 +1,18 @@
+import pytest
```

Comment:
I think we could put this in tests/pagai/conftest.py to avoid having it twice

Reply:
nope sorry